### PR TITLE
variant of collapse

### PR DESCRIPTION
**Pull request by nfranczak**
This PR adds a variant of v-collapse such that there is no border and there is no background color

### CODE REVIEW

File: src/stories/collapse.stories.mdx [hunk 1]
```diff
@@ -87,4 +87,25 @@ Toggles visibility of content
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story
+    name='No Background and No Border'
+    args={{
```
Comment (mrloureed) [inline src/stories/collapse.stories.mdx:93]: Forgot to tell you that you need to add `variant: 'secretUI'` to the args

Comment (mrloureed) [reply src/stories/collapse.stories.mdx:93]: I believe we weren't seeing the border as your browser is in dark mode.

File: src/elements/collapse.svelte [hunk 2]
```diff
@@ -9,6 +9,7 @@ export let title = '';
 export let open = false;
 
 let root: HTMLElement;
+let variant = 'secretUI'
```
Comment (mrloureed) [inline src/elements/collapse.svelte]: Oh maybe change the name to something like `noBorderBackground` or something more descriptive

File: src/stories/collapse.stories.mdx (hunk 1 above)
Comment (nfranczak) [reply src/stories/collapse.stories.mdx:93]: added

File: src/elements/collapse.svelte (hunk 2 above)
Comment (nfranczak) [reply src/elements/collapse.svelte]: changed

File: src/elements/collapse.svelte [hunk 3]
```diff
@@ -8,9 +8,11 @@ import { addStyles, dispatch } from '../lib/index';
 export let title = '';
 export let open = false;
 
-let root: HTMLElement;
+type Variants = 'noBorderBackground' | 'default'
```
Comment (micheal-parks) [inline src/elements/collapse.svelte:11]: Recommend renaming this to `borderless`

Comment (micheal-parks) [reply src/elements/collapse.svelte:11]: or since this cover border + background maybe `minimal`?

File: src/elements/collapse.svelte [hunk 4]
```diff
@@ -23,12 +25,16 @@ const handleClick = (event: Event) => {
...
 
+
+
 <div
   bind:this={root}
   class='relative w-full overflow-hidden'
 >
   <div
-    class='w-full py-2 px-4 flex flex-reverse items-center justify-between border text-black border-black bg-white cursor-pointer'
+    class={cx('w-full py-2 px-4 flex flex-reverse items-center justify-between border text-black border-black bg-white cursor-pointer', {
```
Comment (micheal-parks) [inline src/elements/collapse.svelte:35]: Giving the same rec that I just messaged to jason:
```
<div
    class={cx('w-full py-2 px-4 flex flex-reverse items-center justify-between text-black cursor-pointer', {
      'border border-black bg-white': variant === 'default',
    })},

File: src/stories/collapse.stories.mdx [hunk 5]
```diff
@@ -8,6 +8,10 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs'
     },
   }}
   argTypes={{
+    variant: {
+      description: 'Theses are the variant options.',
```
Comment (micheal-parks) [inline src/stories/collapse.stories.mdx:12]: We should describe here that the default variant adds border and background

**Review by mrloureed — APPROVED** [review]
LGTM

**Pull request merged**